azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageListView.kt

```diff
@@ -41,6 +42,7 @@ internal fun MessageListView(
     dispatchers: Dispatch
 ) {
     requestPages(scrollState, messages, dispatchers)
+    sendReadReceipt(scrollState, messages, dispatchers)
```

Comment:
if(messages.isNotEmpty()) {
        sendReadReceipt(scrollState, messages, dispatchers)
    }

    Will solve the runtime error.

Reply:
thx, I addressed it and updated code.